Doesn't compile without `-Wno-implicit-function-declaration`

Hi, I was packaging `pwait` for https://github.com/pkgforge/soarpkgs
The default build flags error out with:
```bash
/tmp/tmp.dmRJRUh1zL/TEMPREPO/src/ptrace.c:194:9: error: call to undeclared function 'geteuid'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  194 |     if (geteuid() != 0) {
      |         ^
1 error generated.
```

If I pass `-Wno-implicit-function-declaration`, it compiles fine.
I couldn't find this documented in the wiki, if this is already known.

--- a/src/ptrace.c
+++ b/src/ptrace.c
@@ -5,6 +5,7 @@
 #include <stdlib.h>
 #include <sysexits.h>
 #include <syslog.h>
+#include <unistd.h>
 #include "pwait.h"
 
 /* When the tracee is about to exit, waitpid returns a status

